AUTHORS.txt

```diff
@@ -17,6 +17,8 @@ Jonathan Müller <jonathan@fotokite.com>
 Kevin Caffrey <kcaffrey@gmail.com>
 Maksim Nesterov <msnesterov@avito.ru>
 Mathis Engelbart <mathis.engelbart@gmail.com>
+Miroslav <sedivy.miro@gmail.com>
```

Comment:
Do you mind fixing the commit author, so you appear only once in the list?

Reply:
I am curious why it generated me twice in a first place. Maybe it did not recognize character `Š` correctly and then repeated it without my last name.